apis/kueue/webhooks/clusterqueue_webhook.go

```diff
@@ -105,7 +101,6 @@ func ValidateClusterQueue(cq *kueue.ClusterQueue) field.ErrorList {
 		allErrs = append(allErrs, validateNameReference(cq.Spec.Cohort, path.Child("cohort"))...)
 	}
 	allErrs = append(allErrs, validateResources(cq.Spec.Resources, path.Child("resources"))...)
-	allErrs = append(allErrs, validateQueueingStrategy(string(cq.Spec.QueueingStrategy), path.Child("queueingStrategy"))...)
```

Comment:
How? I didn't see any API maker about this.

Reply:
`// +kubebuilder:validation:Enum=StrictFIFO;BestEffortFIFO` could do this.

Reply:
Yes, I just mean we didn't add this marker.

Reply:
Actually we already have this marker
https://github.com/kubernetes-sigs/kueue/blob/d1d445f16478ba2740cf4cd810acfd27e3366d20/apis/kueue/v1alpha2/clusterqueue_types.go#L137

Reply:
Got that, it's a kind of cleanup. Thanks.